### DISCUSSION TITLE
2023-11-04

### DISCUSSION THREAD

**Discussion by ngsy** [10-rolling/rollingPolling#15] [Daily_Meeting]
### ✅ 어제 해낸 일

- [x] Directory  구조 설계
  > [Directory](https://www.notion.so/Directory-9066936c296e43f386c9b2b60124f290)

- [x]  styled-components 네이밍 컨벤션 확정 
  > Styled 시작한다 ex) styledButton 

- [x] branch protection rules 설정 
  > PR 2명이상 Approve 후 merge 

- [x] 팀 회의 시간  및 회의록 작성 순서 결정 
  > 매일 오전 10시 ,토요일 11시 그외 추가회의 필요시 유기적으로 결정
  > 수영님 > 수님 > 슬희님 > 윤혁님

- [x] 협업 규칙 결정 
  > 1.협업 관련한 메세지는  디스코드 (쓰레드 - 프로젝트 이야기)
  > 2.메세지 확인시 이모지 남기기
  > 3.기간 내에 개발 못하면 비난을 주자 (강하게)
  > 4.하면서 모르는 거 있으면 바로바로 물어보기
  > 5.서로 개발 상황 자주 공유하기(git milestone 활용)
  > 6.일단 해보자

#### 개발하기 전에 github flow 및 동작 과정에 대해 명시적으로 정리하면 좋을꺼같음 
  1.새로운 기능 추가 할때
   > issues 등록  -> (라벨/) + (#이슈번호) 브랜치 생성 -> 개발 -> 커밋 & 푸시 

  2.remote 레포에서 로컬 레포로 pull 할때
> 별도 라이브러리 사용하지않으면 git pull 로 하면됌 
> main 브랜치는 항상 최신상태를 유 지하기때문

#### zustand , react-query  를 어떻게 활용할지?
   1. zustand , react-query에 대한 정보 
> 다음주 월요일 자료조사 및 정보 공유 
> 현재 디렉토리에 어떻게 적용시킬지 구조?
> zustand : 윤혁님  , 수영님
> react-query: 슬희님  , 수님 

   2. 활용 방법 
> 다음주 월요일 결정 

#### rebase 활용방법 
 > 1. 예시 상황 a, b 브랜치로 하나의 공통컴포넌트 작업중 
 > 2. a 브랜치 b 브랜치 중 a 브랜치가 먼저 merge를 하여 b브랜치 merge시 충돌 가능성 및 충돌 발생하면
 > 3. a 브랜치가 merge한 시점을 rebase 하여 b브랜치를 merge한다.

### 💪🏻 오늘 할일

공통 컴포넌트 개발 

1 ) Text Field > 수영님

2 ) Button > 수님

3 ) Toast + 에디터 > 윤혁님

4 ) Badge + Option > 슬희님

### 🤔 특이사항

10팀 화이팅!

https://github.com/10-rolling/rollingPolling/assets/63100352/bce3f906-1269-4470-95b9-1c38fd7e233d

![KakaoTalk_Photo_2023-11-04-16-13-48](https://github.com/10-rolling/rollingPolling/assets/72487120/8d69138d-e448-47ac-9fca-de3d00563905)

**Comment by oauch** [comment]
제가 봐도 저 대단하게(?) 생겼네요 🫠

**Reply by ngsy** [reply]
잉  분명 답글했는데..안올라갔나보네용 
태이님👍

**Comment by hanseulhee** [comment]
사진에서 술 냄새 나요

**Reply by ngsy** [reply]
ㅋㅋㅋㅋㅋㅋ🍾🍻🍻 짠~!
